src/cmds/core/other.py

```diff
@@ -1,14 +1,31 @@
 import logging
 
-from discord import ApplicationContext, Embed, Interaction, Message, WebhookMessage, slash_command
+from discord import ApplicationContext, Embed, Interaction, Message, WebhookMessage, slash_command, ui
 from discord.ext import commands
 
 from src.bot import Bot
 from src.core import settings
 
+from slack_webhook import Slack
+
 logger = logging.getLogger(__name__)
 
 
+class FeedbackModal(ui.Modal):
+    def __init__(self, *args, **kwargs) -> None:
+        super().__init__(*args, **kwargs)
+
+        self.add_item(discord.ui.InputText(label="Title"))
+        self.add_item(discord.ui.InputText(label="Feedback", style=discord.InputTextStyle.long))
+    async def callback(self, interaction: discord.Interaction):
+        await interaction.response.send_message("Thank you, your feedback has been recorded.")
+        slack = Slack(url=settings.slack_webhook)
+
+        slack.post(text="New Feedback", attachments = [{
+            "title": self.children[0].value,
+            "text": value=self.children[1].value
```

Comment:
This is a mistake, you cannot assign to value and assign that value to "text" key.

Reply:
Was the way I had used this library previously- N/A since will port to proper slack library.